src/cli/cli.cpp

```diff
@@ -5845,6 +5898,16 @@ template <> otError Interpreter::Process<Cmd("promiscuous")>(Arg aArgs[])
 {
     otError error = OT_ERROR_NONE;
 
+    /**
+     *@cli promiscuous
+     *@code
+     *promiscuous
+     *Disabled
+     *Done
+     *@endcode
+     *@par api_copy
```

Comment:
I would do `@sa` for both APIs being used here and then have a custom description after a `@par`

otLinkIsPromiscuous
otPlatRadioGetPromiscuous